src/translations/en.json

```diff
@@ -2440,6 +2440,7 @@
             "configure": "Configure",
             "system_options": "System options",
             "documentation": "Documentation",
+            "download_diagnostics": "Download diagnostics",
```

Comment:
Should this be more specific?

Reply:
I don't think so, you're in the menu of a specific config entry

Reply:
It make sense to me and you, but when we eventually get integration/device specific diagnostics, it might be confusing  for the user

Reply:
I think that if we get there, we should just make this open a diagnostics dialog with the different options.

Reply:
That works 👍